internal/template.go

```diff
@@ -49,21 +49,33 @@ func ExecuteTemplate(sourceFilePath string, funcMap template.FuncMap, verbose bo
 }
 
 // GetFuncMap builds the relevant function map to helm_ssm
-func GetFuncMap(profile string) template.FuncMap {
+func GetFuncMap(profile string, clean bool) template.FuncMap {
+
+	cleanFunc := func(...interface{}) (string, error) {
+		return "CLEANED_BY_HELM_SSM", nil
```

Comment:
Actually, on second thought, maybe we can have the feature of tagging cleaned stuff under an optional flag, which could be used for debugging, and by default just cleaning it. It would probably make things neater without loss of functionality.

Reply:
Love the extra functionality under another flag 👌 
One thing that I am still "worried-ish" is that we keep the key without value. Meaning that this will do:

```
//before
example: {{ssm \"foo\" | quote | indent 8}}
//after
example:
```

And I would expect:

```
//before
example: {{ssm \"foo\" | quote | indent 8}}
//after
```

Regardless, we can advance with this and see if there is any problem whatsoever, good stuff!